lib/glimesh_web/live/user_live/profile.html.leex

```diff
@@ -50,12 +50,12 @@
 
                     <div class="simple--counter-container mt-4">
 
-                        <div class="counter-container">
+                        <a href="profile/followers"><div class="counter-container">
```

Comment:
Can this be a proper phoenix link so we don't suffer from accidental url munging?

Reply:
Sure